deploy/helm/templates/policies.yaml

```diff
@@ -823,3 +823,7 @@ data:
     kubernetes.namespace]))\n\n\tres := {\n\t\t\"msg\": msg,\n\t\t\"id\": __rego_metadata__.id,\n\t\t\"title\":
     __rego_metadata__.title,\n\t\t\"severity\": __rego_metadata__.severity,\n\t\t\"type\":
     __rego_metadata__.type,\n\t}\n}\n"
+  policy.use_limit_range.kinds: LimitRange
```

Comment:
Can you add these policies to ./hack/starboard-policies-config.sh and use this script to generate the ConfigMap?